src/tests/ftest/server/daos_server_config.yaml

```diff
@@ -0,0 +1,192 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+  test_servers:
+    - server-A
+server_manager:
+    srv_timeout: 30
+timeout: 120
+server_config:
+  name: daos_server
+
+# The 'config_val' list is composed of the following values:
+# - name of the config attribute, used to look for the attribute object
+# - test value that will be inserted into the server config for the attribute
+# - PASS or FAIL value, specifying how the server is expected to function
+# - value specifying if the attribute is part of the general config values or
+#     the servers config values
+server_config_val: !mux
+  name_special_chars:
+    config_val:
+      - "name"
+      - "! @#$%^&*()_+{}|:<>?-=[];',./"
+      - "FAIL"
+      - "gen"
+  name_alphanumeric:
+    config_val:
+      - "name"
+      - "ABCD1234abcd"
+      - "PASS"
+      - "gen"
+  name_numeric:
+    config_val:
+      - "name"
+      - "12345678"
+      - "PASS"
+      - "gen"
+  name_combo:
+    config_val:
+      - "name"
+      - "AB-C&__dal1234"
+      - "PASS"
+      - "gen"
+  name_list:
+    config_val:
+      - "name"
+      - "amjustin, 12345, abcdef"
+      - "FAIL"
+      - "gen"
+  # access_point_repeated:
+  #   config_val:
+  #     - "access_points"
+  #     - "[localhost:10001, localhost:10001]"
+  #     - "FAIL"
+  #     - "gen"
+  # access_point_wrong_port:
```

Comment:
should also add a test for supplying access point host without port, that will be the approved and documented method going forward after https://github.com/daos-stack/daos/pull/1637/ lands (supplying port in access point config file address assignments will not be shown in the example config files and whilst still possible will not be part of normal procedure)

Reply:
Will do, I'll create a ticket for those cases in particular and discuss how to handle them. I'll add a new case to add the accesspoint with no port. Thanks Tom.